src/Homework14/FIFOQueue.java

```diff
@@ -0,0 +1,36 @@
+package Homework14;
+
+import java.util.List;
+
+public class FIFOQueue implements Queue {
```

Comment:
why queue? you have LinkedList with methods which give for you functionality for create